### ISSUE TITLE
forum.codeigniter.com is broken

### ISSUE THREAD

**Issue by negrusti** [codeigniter4/CodeIgniter4#5168]
Hi, your forum is broken for days already, please fix...

**Comment by lonnieezell** [comment]
I think you might have bad luck on your timing. I fixed it a couple of days ago, someone else fixed it this morning. I can fix again when I get home this afternoon. We are in the process of moving servers to what we hope should have a more robust database solution that will stop this from happening since I’ve not yet found a permanent fix for this.

Thanks for your patience

**Comment by negrusti** [comment]
@lonnieezell converting your tables to InnoDB might be a good start

**Comment by truelineinfotech** [comment]
Flarum is good option https://flarum.org/

Just sharing my view.

**Comment by lonnieezell** [comment]
Yup - I like Flarum a lot. I dont know that would fix an issue with a 1TB+ db having tables getting corrupted and needing repair, though, unless Flarum handles it differently. Definitely have a few things to look into, but first is getting onto new hosting, which is in the works. 

I fixed it again. Interestingly enough when I went to visit the site it pulled up fine for me. I tried in a private browser window and it failed so it's possible I hadn't noticed because it was only happening when writing a new session to the db.
